src/main/java/com/github/vaibhavsinha/kong/impl/helper/RetrofitServiceCreator.java

```diff
@@ -1,14 +1,16 @@
 package com.github.vaibhavsinha.kong.impl.helper;
 
 //import com.jakewharton.retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)